Sources/PhotoCaptureViewController/CaptureManager.swift

```diff
@@ -11,17 +11,17 @@ enum CaptureManagerViewfinderMode {
 }
 
 protocol CaptureManagerDelegate: AnyObject {
+    func captureManager(_ manager: CaptureManager, didCaptureImage data: Data?, withMetadata metadata: NSDictionary?)
```

Comment:
Perhaps `didCaptureImageData` is more descriptive since it's an instance of `Data`?